code/SiteConfig.php

```diff
@@ -140,19 +150,29 @@ public function getCMSFields()
                             'data-placeholder',
                             _t('SilverStripe\\CMS\\Model\\SiteTree.GroupPlaceholder', 'Click to select group')
                         ),
+                    $editorMembersField = ListboxField::create(
+                        "EditorMembers",
+                        _t(self::class . '.EDITORMEMBERS', "Editor Users"),
+                        $membersMap,
+                    ),
                     $topLevelCreatorsOptionsField = OptionsetField::create(
                         "CanCreateTopLevelType",
                         _t(self::class . '.TOPLEVELCREATE', "Who can create pages in the root of the site?")
                     ),
                     $topLevelCreatorsGroupsField = ListboxField::create(
                         "CreateTopLevelGroups",
-                        _t(self::class . '.TOPLEVELCREATORGROUPS', "Top level creators")
+                        _t(self::class . '.TOPLEVELCREATORGROUPS2', "Top level creator groups")
```

Comment:
The "2" is intentional - we're changing the wording of the string, which means we also have to change the key.
If we don't change the key, the following happens:
1. When we run translations, we'll override the wording used in 4.13 and 5.1
2. Any translations for this key will be wrong, because they're translations of the _old_ string, not the new one.